scripts/brands.js

```diff
@@ -0,0 +1,108 @@
+import request from 'request'
+import { includes } from 'lodash'
+import ProgressBar from 'progress'
+
+const NOT_KONNNECTORS = [
+  'libs',
+  'konitor',
+  'docs',
+  'cozy-konnector-extension',
+  'renovate-config-cozy-konnector',
+]
+
+const getRequestOptions = url => ({
+  url,
+  headers: {
+    'User-Agent': 'request'
+  }
+})
+
+const githubUrl = url => {
+  return new Promise((resolve, reject) => {
+    request.get(getRequestOptions(url), (error, response, body) => {
+      if (error) {
+        reject(error)
+      }
+      resolve(body)
+    })
+  })
+}
+
+const githubJsonUrl = async url => {
+  const body = await githubUrl(url)
+  return JSON.parse(body)
+}
+
+const getRepos = limit => githubJsonUrl(`https://api.github.com/orgs/konnectors/repos?per_page=${limit}`)
+const repoFilter = repos => repos
+  .filter(repo => !includes(NOT_KONNNECTORS, repo.name))
+  .map(repo => repo.name)
+const getManifest = repo => githubJsonUrl(`https://raw.githubusercontent.com/konnectors/${repo}/build/manifest.konnector`)
+const getIndex = repo => githubUrl(`https://raw.githubusercontent.com/konnectors/${repo}/build/index.js`)
+
+const getIdentifiers = async repo => {
```

Comment:
I think adding some comments to this function would help understand what it does. that's not obvious at first sight

Reply:
I add more comments ;)